.github/tests/lm_tests.py

```diff
@@ -2,19 +2,28 @@
 import pytest
 
 import lotus
-from lotus.models import OpenAIModel
+from lotus.models import LM
 
 # Set logger level to DEBUG
 lotus.logger.setLevel("DEBUG")
 
 
+@pytest.fixture
+
 @pytest.fixture
 def setup_models():
     # Setup GPT models
-    gpt_4o_mini = OpenAIModel(model="gpt-4o-mini")
-    gpt_4o = OpenAIModel(model="gpt-4o")
+    gpt_4o_mini = LM(model="gpt-4o-mini")
+    gpt_4o = LM(model="gpt-4o")
     return gpt_4o_mini, gpt_4o
 
+def test_lm_initialization():
```

Comment:
I'm seeing that the other tests fail, even after I add `max_batch_size` to prevent runtime errors. Can you check on your end that all the other tests pass, and patch it if they fail? You can run the tests with `pytest .github/tests/lm_tests.py`

Reply:
It would also be good to just have some basic sanity checks that just checks the input/output of the `LM` to make sure its returning what we expect.